dts/bindings/counter/nxp,lptmr.yaml

```diff
@@ -34,3 +34,14 @@ properties:
     type: int
     required: true
     description: Represents the width of the timer in bits.
+
+  counter-mode-enable:
```

Comment:
This property should be `pulse-mode-en` if the most common case is going to be counter mode.

Reply:
alternatively, it might make more sense to make it a string enum rather than a boolean, since this isnt really enabling a feature but rather a selection of a mode, it makes it much more clear that way